src/libraries/System.DirectoryServices/src/System.DirectoryServices.csproj

```diff
@@ -3,7 +3,7 @@
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent);$(NetCoreAppPrevious)-windows;$(NetCoreAppPrevious);$(NetCoreAppMinimum)-windows;$(NetCoreAppMinimum);netstandard2.0</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IncludeInternalObsoleteAttribute>true</IncludeInternalObsoleteAttribute>
-    <NoWarn>$(NoWarn);IDE0059;IDE0060;CA1822</NoWarn>
+    <NoWarn>$(NoWarn);IDE0059;IDE0060;CA1822;CP0001</NoWarn>
```

Comment:
This CP0001 NoWarn can't easily be removed here with a suppression file. There would dozens of entries needed due to the type forwards..

Reply:
Agree with Viktor -- we should *never* NoWarn a CP* warning in dotnet/runtime.  @ViktorHofer -- perhaps we need an error to prevent developers from doing this.

I don't think you should even have any compatibility warnings since these types have existed forever we're just moving them down and hiding the dependency from packaging - however API Compat will still see it here.  Please fix my suggestion for S.S.P and then remove this and add any suppressions you still see.

Reply:
The only other suppression for CP0001 in a csproj is for DI: https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.DependencyInjection/src/Microsoft.Extensions.DependencyInjection.csproj#L8

I can take a look at that after this PR.